translations/tiled_de.ts

```diff
@@ -5126,7 +5126,7 @@ Bitte Format angeben.</translation>
         <location filename="../src/tiled/transformmapobjects.cpp" line="+83"/>
         <source>Move %n Object(s)</source>
         <translation>
-            <numerusform>Objekt verschieben</numerusform>
+            <numerusform>%n Objekt verschieben</numerusform>
```

Comment:
@Atalanttore Note that it's not really needed to add the `%n` in this case, since it is always 1 in German, and "Objekt verschieben" already makes it clear that it is about a single object. I'll merge this for now, but it's something to keep in mind when translating plural forms. :-)